network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -151,15 +151,15 @@ <h3 class="pni-product-smaller-body-bold mb-2">{% trans "What data does it colle
 
 
           {% trans "How does it use this data?" as how_does_it_use_data_collected %}
-          {% include "fragments/product_criterion.html"  with  label=how_does_it_use_data_collected  value=product.how_does_it_use_data_collected  %}
+          {% include "fragments/product_criterion.html"  with  label=how_does_it_use_data_collected  value=product.how_does_it_use_data_collected  ding_control=product.data_collection_policy_is_bad %}
```

Comment:
Hm, can we change this name to `show_ding` or even just `ding`? A controller is typically far more elaborate than a boolean.